health/micro-ui/web/packages/modules/campaign-manager/src/pages/employee/NewCampaignCreate/CampaignHome.js

```
@@ -152,14 +154,25 @@ const CampaignHome = () => {
+        >
+          <CopyAll />
+          <div className={"descStyle"}>{t("HCM_IMPORT_EXISTING_CAMPAIGN")}</div>
+        </div>
         <div
           className={"cardStyle disabledCard"}
           onClick={() => {
-            // Add functionality for importing existing campaign
+            // Add functionality for campaign templates
           }}
+          tabIndex={2}
         >
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)